ios/Handlers/RNGestureHandler.m

```diff
@@ -356,12 +371,25 @@ - (BOOL)hasCustomCriteria
 {
     return !isnan(_minDistSq) || !isnan(_minDeltaX) || !isnan(_minDeltaY)
         || !isnan(_minOffsetX) || !isnan(_minOffsetY)
-        || !isnan(_minVelocityX) || !isnan(_minVelocityY) || !isnan(_minVelocitySq);
+        || !isnan(_minVelocityX) || !isnan(_minVelocityY) || !isnan(_minVelocitySq)
+        || !isnan(_maxDeltaY);
```

Comment:
I think setting "max" bounds shouldn't affect `_hasCustomCriteria` as when you for example only set "max" but no min, then gesture will never fire. The purpose of `_hasCustomCriteria` was so that we can use platform default settings of `UIPanGestureHandler` if no other criteria are set, but the way it works is that if some "min" constrains are set we have a custom logic that is responsible for testing if handler should activate. So if there are no "min" props set and `hasCustomCriteria` is true the gesture will never activate.